.bumpversion.cfg

```diff
@@ -17,3 +17,7 @@ replace = pkgver={new_version}
 [bumpversion:file:pkg/openwrt/Makefile]
 search = PKG_VERSION:={current_version}
 replace = PKG_VERSION:={new_version}
+
+[bumpversion:file:ustreamer.1]
+search = version "{current_version}"
```

Comment:
Mus be `"version {current_version}"` I guess. Same to replace.

Reply:
Yup, that looks to work

```
diff --git a/ustreamer.1 b/ustreamer.1
index fcd7b51..e86d4d2 100644
--- a/ustreamer.1
+++ b/ustreamer.1
@@ -1,6 +1,6 @@
 .\" Manpage for ustreamer.
 .\" Open an issue or pull request to https://github.com/pikvm/ustreamer to correct errors or typos
-.TH USTREAMER 1 "version 2.2" "November 2020"
+.TH USTREAMER 1 "version 2.3" "November 2020"
```